environments/nonprod/variables.tf

```diff
@@ -190,3 +190,27 @@ variable "monitoring_centers" {
   description = "Input value for the centralized monitoring projects"
   default     = {}
 }
+
+variable "network_self_links" {
```

Comment:
Thanks Jackson,  in 299 I missed upsourcing one of the files for the non prod zone - only the prod zone is in use - the dns.tf stub should have been removed.